ironfish-cli/src/commands/wallet/rename.ts

```diff
@@ -1,33 +1,33 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Args } from '@oclif/core'
 import { IronfishCommand } from '../../command'
 import { RemoteFlags } from '../../flags'
 
 export class RenameCommand extends IronfishCommand {
   static description = 'Change the name of an account'
 
-  static args = [
-    {
-      name: 'account',
+  static args = {
+    account: Args.string({
       required: true,
       description: 'Name of the account to rename',
-    },
-    {
-      name: 'new-name',
+    }),
+    // TODO(mat): Do we need to do 'new-name', or does oclif do some magic like flags now?
```

Comment:
It looks like args accept a 'name' field, but it doesn't seem to do anything 🤔 

FWIW the name of the arg only affects the help text and the name of the variable in the code, so I think we could rename this to `newName` if we wanted to without meaningfully impacting users

Reply:
My guess is the name field is a derelict from the old version of args that they kept around for.. backwards compatibility? That is kind of strange since it's now decided by the key name. 

It did completely slip my mind that these are _args_ and thus the name is sort of arbitrary - thanks for pointing that out. 😂